src/renderer/components/Input/InputFee.vue

```diff
@@ -203,50 +224,28 @@ export default {
       if (this.fee < this.feeChoices.AVERAGE) {
         return this.$t('INPUT_FEE.LOW_FEE_NOTICE')
       }
-
       return null
-    },
-    isStaticFee () {
-      if (this.feeChoices.MAXIMUM === this.feeChoices.AVERAGE) {
-        return +this.fee === this.feeChoices.AVERAGE
-      }
-
-      return false
-    },
-    isAdvancedFee () {
-      return this.feeChoice === 'ADVANCED'
     }
   },
 
   created () {
-    this.prepareFeeStatistics()
-  },
-
-  methods: {
-    /**
-     * Even if the network provides a fees higher than V1, it will be corrected
-     */
-    prepareFeeStatistics () {
-      let { avgFee, maxFee } = this.$store.getters['network/feeStatisticsByType'](this.transactionType)
-
-      avgFee = avgFee * 1e-8
-      this.$set(this.feeChoices, 'AVERAGE', avgFee < this.maxV1fee ? avgFee : this.maxV1fee)
+    // Fees should be synchronized only when this component is active
+    this.$synchronizer.focus('fees')
```

Comment:
I think the `call` method is enough, these values are not updated with this frequency:

```suggestion
    this.$synchronizer.call('fees')
```